### PR TITLE
Move utility classes to utilities.scss

### LINKED ISSUE #6500: [SCSS] Move utility classes to utilities.scss
Utility classes like `.bg-white`, `.bg-black`, `.bg-gray`, ... etc should be moved into utilities.scss.

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #6500 

Note: PR https://github.com/mozilla/foundation.mozilla.org/pull/6514 removes `source/sass/views/initiatives.scss` so we should wait for that PR to merge first and rebase/update this PR accordingly after.

### CODE REVIEW

File: source/sass/views/initiatives.scss [hunk 1]
```diff
@@ -1,7 +1,3 @@
-.bg-white {
```
Comment (mmmavis) [inline source/sass/views/initiatives.scss:1]: Bootstrap has `.bg-white` as an utility class already: https://getbootstrap.com/docs/4.0/utilities/colors/#background-color

File: source/sass/utilities.scss [hunk 2]
```diff
@@ -7,10 +7,6 @@
   flex: 1;
 }
 
-.position-relative {
```
Comment (mmmavis) [inline source/sass/utilities.scss:10]: Bootstrap has this already: https://github.com/twbs/bootstrap/blob/v4.6.0/scss/utilities/_position.scss#L4-L6

File: source/sass/global.scss [hunk 3]
```diff
@@ -149,7 +149,3 @@ hr {
     }
   }
 }
-
```
Comment (mmmavis) [inline source/sass/global.scss:152]: moved to source/sass/utilities.scss

File: source/sass/utilities.scss (hunk 2 above)
Comment (Pomax) [reply source/sass/utilities.scss:10]: ❗

**Review by Pomax — APPROVED** [review]

**Pull request merged**